lib/core/constants.js

```diff
@@ -31,7 +31,9 @@
 		results: [],
 		resultGroups: [],
 		resultGroupMap: {},
-		impact: Object.freeze(['minor', 'moderate', 'serious', 'critical'])
+		impact: Object.freeze(['minor', 'moderate', 'serious', 'critical']),
+		preloadAssets: Object.freeze(['cssom']),
```

Comment:
What is this for? Why do we need to `.freeze()` it?

Maybe add a comment here explaining "why" we need this (frozen) array?